tests/api_resources/test_customer.py

```diff
@@ -45,21 +46,6 @@ def test_is_modifiable(self, request_mock):
         )
         assert isinstance(resource, stripe.Customer)
 
-    def test_is_deletable(self, request_mock):
-        resource = stripe.Customer.retrieve(TEST_RESOURCE_ID)
-        resource.delete()
-        request_mock.assert_requested(
-            "delete", "/v1/customers/%s" % TEST_RESOURCE_ID
-        )
-        assert resource.deleted is True
-
-    def test_can_delete(self, request_mock):
-        resource = stripe.Customer.delete(TEST_RESOURCE_ID)
-        request_mock.assert_requested(
-            "delete", "/v1/customers/%s" % TEST_RESOURCE_ID
-        )
-        assert resource.deleted is True
```

Comment:
I think you inadvertently removed these tests after a rebase, can you re-add them?

Reply:
Ouch my bad, nice catch. Fixed.